diff --git a/node_modules/@slippi/slippi-js/dist/index.es.js b/node_modules/@slippi/slippi-js/dist/index.es.js
index 9aee259..22a4240 100644
--- a/node_modules/@slippi/slippi-js/dist/index.es.js
+++ b/node_modules/@slippi/slippi-js/dist/index.es.js
@@ -9,6 +9,7 @@ import { Writable } from 'stream';
 import net from 'net';
 import inject from 'reconnect-core';
 import path from 'path';
+import enet from 'enet';
 
 function toHalfwidth(str) {
     // Code reference from https://github.com/sampathsris/ascii-fullwidth-halfwidth-convert
@@ -2583,7 +2584,6 @@ class DolphinConnection extends EventEmitter {
         console.log(`Connecting to: ${ip}:${port}`);
         this.ipAddress = ip;
         this.port = port;
-        const enet = await import('enet');
         // Create the enet client
         const client = enet.createClient({ peers: MAX_PEERS, channels: 3, down: 0, up: 0 }, (err) => {
             if (err) {
