diff --git a/node_modules/@slippi/slippi-js/dist/index.es.js b/node_modules/@slippi/slippi-js/dist/index.es.js
index a3a1430..442be6f 100644
--- a/node_modules/@slippi/slippi-js/dist/index.es.js
+++ b/node_modules/@slippi/slippi-js/dist/index.es.js
@@ -5,6 +5,7 @@ import moment from 'moment';
 import { Writable } from 'stream';
 import { decode, encode } from '@shelacek/ubjson';
 import net from 'net';
+import enet from 'enet';
 import inject from 'reconnect-core';
 import iconv from 'iconv-lite';
 import path from 'path';
@@ -2086,7 +2087,6 @@ class DolphinConnection extends EventEmitter {
         console.log(`Connecting to: ${ip}:${port}`);
         this.ipAddress = ip;
         this.port = port;
-        const enet = await import('enet');
         // Create the enet client
         const client = enet.createClient({ peers: MAX_PEERS, channels: 3, down: 0, up: 0 }, (err) => {
             if (err) {
