diff --git a/node_modules/dotenv-webpack/dist/index.js b/node_modules/dotenv-webpack/dist/index.js
index 8cd15a3..be9b4c6 100644
--- a/node_modules/dotenv-webpack/dist/index.js
+++ b/node_modules/dotenv-webpack/dist/index.js
@@ -182,7 +182,10 @@ var Dotenv = /*#__PURE__*/function () {
         return obj;
       }, {}); // fix in case of missing
 
-      formatted['process.env'] = '{}';
+      // This one line seems to remove all the process.env variables in the main process
+      // causing the main process to throw an 'Env ELECTRON_HMR_SOCKET_PATH is not set' error.
+      // So we're just gonna comment it out.
+      // formatted['process.env'] = '{}';
       return formatted;
     }
     /**
