diff --git a/node_modules/electron-webpack/out/targets/RendererTarget.js b/node_modules/electron-webpack/out/targets/RendererTarget.js
index a74e416..fc13d97 100644
--- a/node_modules/electron-webpack/out/targets/RendererTarget.js
+++ b/node_modules/electron-webpack/out/targets/RendererTarget.js
@@ -124,7 +124,7 @@ class BaseRendererTarget extends _BaseTarget().BaseTarget {
       test: /\.s([ac])ss$/,
       use: cssHotLoader.concat("sass-loader")
     }, {
-      test: /\.(png|jpe?g|gif|svg)(\?.*)?$/,
+      test: /\.(png|jpe?g|gif)(\?.*)?$/,
       use: {
         loader: "url-loader",
         options: (0, _BaseTarget().configureFileLoader)("imgs")
