diff --git a/node_modules/kysely/dist/cjs/dialect/sqlite/sqlite-dialect-config.d.ts b/node_modules/kysely/dist/cjs/dialect/sqlite/sqlite-dialect-config.d.ts
index a873682..b446ad7 100644
--- a/node_modules/kysely/dist/cjs/dialect/sqlite/sqlite-dialect-config.d.ts
+++ b/node_modules/kysely/dist/cjs/dialect/sqlite/sqlite-dialect-config.d.ts
@@ -32,10 +32,10 @@ export interface SqliteDatabase {
 }
 export interface SqliteStatement {
     readonly reader: boolean;
-    all(parameters: ReadonlyArray<unknown>): unknown[];
-    run(parameters: ReadonlyArray<unknown>): {
+    all(...parameters: unknown[]): unknown[];
+    run(...parameters: unknown[]): {
         changes: number | bigint;
         lastInsertRowid: number | bigint;
     };
-    iterate(parameters: ReadonlyArray<unknown>): IterableIterator<unknown>;
+    iterate(...parameters: unknown[]): IterableIterator<unknown>;
 }
diff --git a/node_modules/kysely/dist/cjs/dialect/sqlite/sqlite-driver.js b/node_modules/kysely/dist/cjs/dialect/sqlite/sqlite-driver.js
index 9f02da6..69b855a 100644
--- a/node_modules/kysely/dist/cjs/dialect/sqlite/sqlite-driver.js
+++ b/node_modules/kysely/dist/cjs/dialect/sqlite/sqlite-driver.js
@@ -65,10 +65,10 @@ class SqliteConnection {
         const stmt = this.#db.prepare(sql);
         if (stmt.reader) {
             return Promise.resolve({
-                rows: stmt.all(parameters),
+                rows: stmt.all(...parameters),
             });
         }
-        const { changes, lastInsertRowid } = stmt.run(parameters);
+        const { changes, lastInsertRowid } = stmt.run(...parameters);
         return Promise.resolve({
             numAffectedRows: changes !== undefined && changes !== null ? BigInt(changes) : undefined,
             insertId: lastInsertRowid !== undefined && lastInsertRowid !== null
@@ -81,7 +81,7 @@ class SqliteConnection {
         const { sql, parameters, query } = compiledQuery;
         const stmt = this.#db.prepare(sql);
         if (select_query_node_js_1.SelectQueryNode.is(query)) {
-            const iter = stmt.iterate(parameters);
+            const iter = stmt.iterate(...parameters);
             for (const row of iter) {
                 yield {
                     rows: [row],
diff --git a/node_modules/kysely/dist/esm/dialect/sqlite/sqlite-dialect-config.d.ts b/node_modules/kysely/dist/esm/dialect/sqlite/sqlite-dialect-config.d.ts
index a873682..b446ad7 100644
--- a/node_modules/kysely/dist/esm/dialect/sqlite/sqlite-dialect-config.d.ts
+++ b/node_modules/kysely/dist/esm/dialect/sqlite/sqlite-dialect-config.d.ts
@@ -32,10 +32,10 @@ export interface SqliteDatabase {
 }
 export interface SqliteStatement {
     readonly reader: boolean;
-    all(parameters: ReadonlyArray<unknown>): unknown[];
-    run(parameters: ReadonlyArray<unknown>): {
+    all(...parameters: unknown[]): unknown[];
+    run(...parameters: unknown[]): {
         changes: number | bigint;
         lastInsertRowid: number | bigint;
     };
-    iterate(parameters: ReadonlyArray<unknown>): IterableIterator<unknown>;
+    iterate(...parameters: unknown[]): IterableIterator<unknown>;
 }
diff --git a/node_modules/kysely/dist/esm/dialect/sqlite/sqlite-driver.js b/node_modules/kysely/dist/esm/dialect/sqlite/sqlite-driver.js
index 7b5566d..f98266d 100644
--- a/node_modules/kysely/dist/esm/dialect/sqlite/sqlite-driver.js
+++ b/node_modules/kysely/dist/esm/dialect/sqlite/sqlite-driver.js
@@ -62,10 +62,10 @@ class SqliteConnection {
         const stmt = this.#db.prepare(sql);
         if (stmt.reader) {
             return Promise.resolve({
-                rows: stmt.all(parameters),
+                rows: stmt.all(...parameters),
             });
         }
-        const { changes, lastInsertRowid } = stmt.run(parameters);
+        const { changes, lastInsertRowid } = stmt.run(...parameters);
         return Promise.resolve({
             numAffectedRows: changes !== undefined && changes !== null ? BigInt(changes) : undefined,
             insertId: lastInsertRowid !== undefined && lastInsertRowid !== null
@@ -78,7 +78,7 @@ class SqliteConnection {
         const { sql, parameters, query } = compiledQuery;
         const stmt = this.#db.prepare(sql);
         if (SelectQueryNode.is(query)) {
-            const iter = stmt.iterate(parameters);
+            const iter = stmt.iterate(...parameters);
             for (const row of iter) {
                 yield {
                     rows: [row],
